app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -514,6 +515,12 @@ class ChatRoomPresenter @Inject constructor(
                     val fileSize = uriInteractor.getFileSize(uri)
                     val maxFileSizeAllowed = settings.uploadMaxFileSize()
 
+                    Timber.d("uri : ${uri.toString()}")
```

Comment:
@Shailesh351 Did you mean to keep this logging?  If so can you make it more descriptive, or remove them please...